internal/db/db_certificate_authorities_test.go

```diff
@@ -25,7 +25,7 @@ func TestRootCertificateAuthorityEndToEnd(t *testing.T) {
 		t.Fatalf("CA found when no CA's should be available")
 	}
 
-	caID, err := database.CreateCertificateAuthority(RootCACSR, RootCAPrivateKey, RootCACRL, RootCACertificate+"\n"+RootCACertificate)
+	caID, err := database.CreateCertificateAuthority(RootCACSR, RootCAPrivateKey, RootCACRL, RootCACertificate+"\n"+RootCACertificate, 0)
```

Comment:
Does a user with the ID `0` exist? Please see related comment, this might not be an issue.

Reply:
This was fixed in b15ae23871a1735449ef189532fccc6b646101f5